rest-api-spec/src/main/resources/rest-api-spec/api/reindex.rethrottle.json

```diff
@@ -0,0 +1,24 @@
+{
+  "reindex.rethrottle": {
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/plugins/master/plugins-reindex.html",
+    "methods": ["POST"],
+    "url": {
+      "path": "/_reindex/{task_id}/_rethrottle",
+      "paths": ["/_reindex/{task_id}/_rethrottle", "/_update_by_query/{task_id}/_rethrottle"],
+      "parts": {
+        "task_id": {
+          "type": "string",
+          "description": "The task id to rethrottle"
+        }
+      },
+      "params": {
+        "requests_per_second": {
+          "type": "float",
+          "default": 0,
```

Comment:
I'm not sure what that'll do. I wrote the default that way to mean "if you don't send the parameter at all it'll default to unlimited." I've not tried sending the parameter without a value - probably an exception parsing the float.